README.md

```diff
@@ -24,6 +24,15 @@ This version of the SDK doesn't have support for Amazon Store. If you would like
 📮 | Great support - [Help Center](https://revenuecat.zendesk.com) 
 
 ## Getting Started
+There are 2 supported mechanisms to use `purchases-unity`:
```

Comment:
We don't usually add this in the README of each repo. Instead we update the docs directly. I guess that's the preferred approach right? In any case, I thought I would add this here and can move to the docs later instead after it's been reviewed.

Reply:
mixed feelings, honestly. On one side I feel like it doesn't hurt to overdocument, but in reality it _can_, since it's yet another place to maintain.

Reply:
let's get @codykerns's thoughts on this